### PR TITLE
Minor Grammar

### PR DESCRIPTION
**Pull request by danbaker7**
grammar

### CODE REVIEW

File: articles/cloud-storage/cs-how-install-cifs-ecs.md [hunk 1]
```diff
@@ -94,10 +94,10 @@ CIFS-ECS offers a single caching option:
 
 11. Click **OK** to return to the *Settings* page.
 
-12. The rest of the fields on the *Settings* page enable you to configure the root directory, encryption and compression settings.
+12. The rest of the fields on the *Settings* page enable you to the root directory, encryption and compression settings.
```
Comment (vrush20) [inline articles/cloud-storage/cs-how-install-cifs-ecs.md]: change back pls

Comment (danbaker7) [reply articles/cloud-storage/cs-how-install-cifs-ecs.md]: yeah I got hasty. But I fixed it

File: articles/cloud-storage/cs-how-install-cifs-ecs.md [hunk 2]
```diff
@@ -94,10 +94,10 @@ CIFS-ECS offers a single caching option:
 
 11. Click **OK** to return to the *Settings* page.
 
-12. The rest of the fields on the *Settings* page enable you to configure the root directory, encryption and compression settings.
```
Comment (suehighmoor) [inline articles/cloud-storage/cs-how-install-cifs-ecs.md]: Original was correct (enable not enables because you are referring to multiple fields)

Comment (danbaker7) [reply articles/cloud-storage/cs-how-install-cifs-ecs.md]: Yes you're right, apologies.

File: articles/cloud-storage/cs-how-install-cifs-ecs.md [hunk 3]
```diff
@@ -111,14 +111,14 @@ CIFS-ECS offers a single caching option:
 
     ![Dashboard of CIFS-ECS](images/cs-ecs-installtion-step-four.png)
 
-17. You can share the CIFS-ECS drive, or a folder within it, so that other machines on the network can access it.
+17. You can share the CIFS-ECS drive, or a folder within it so that other machines on the network can access it.
```
Comment (boardfish) [inline articles/cloud-storage/cs-how-install-cifs-ecs.md]: Original seems correct in this instance.

**Pull request closed without merging**